scripts/build_postgres.sh

```diff
@@ -0,0 +1,37 @@
+# Build libpq on macOS or Linux
+
+if [ ! -d "postgres-${POSTGRES_TAG}/" ]; then
+    curl -sL \
+        https://github.com/postgres/postgres/archive/${POSTGRES_TAG}.tar.gz \
+        | tar xzf -
+
+    cd "postgres-${POSTGRES_TAG}/"
+
+    if [ $(uname) == 'Linux' ]; then
```

Comment:
I really, really, prefer to have separate scripts rather than a more complex one peppered with ifs.

Reply:
OK, can do, but for the record, that's one `if` and the alternative is 12 statements-worth of code duplication.